packages/design-system-twrnc-preset/babel.config.js

```diff
@@ -4,7 +4,12 @@ module.exports = {
       presets: [
         'module:metro-react-native-babel-preset',
         '@babel/preset-env',
-        '@babel/preset-react',
+        [
+          '@babel/preset-react',
+          {
+            runtime: 'automatic',
+          },
+        ],
```

Comment:
Fixes this warning and aligns with design-system-react-native https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-system-react-native/babel.config.js#L8-L11

<img width="789" alt="Screenshot 2024-11-18 at 3 58 10 PM" src="https://github.com/user-attachments/assets/089820bc-0a8f-41ea-ba7d-72353f14cfd1">